Cargo.toml

```diff
@@ -19,6 +19,7 @@ base64 = "0.21.0"
 byteorder = "1.4.3"
 crc32fast = "1.3.2"
 once_cell = "1.10.0"
+madvise = { version="0.1", optional= true }
```

Comment:
Note that `memmap2` has integrated support for `madvise(2)`: https://docs.rs/memmap2/latest/memmap2/struct.Mmap.html#method.advise

Reply:
Yes good point, let's use that and remove the feature flag

Reply:
Ah thank you!